core/model/models/volume-dataset.js

```diff
@@ -259,12 +259,7 @@ exports.VolumeDataset = Montage.specialize({
                 id: 'ui/controls/viewer.reel'
             },
             collectionNameExpression: "'Datasets'",
-            nameExpression: "name",
-            iconValueMapping: {
-                "FILESYSTEM": "ui/icons/directory.reel",
-                "VOLUME": "ui/icons/volume.reel"
-            },
-            iconValueExpression: "type"
+            nameExpression: "name"
```

Comment:
I am not sure this has anything to do with the container UI (in fact, I am not event sure we want to do that...)